aiogram/fsm/storage/mongo.py

```diff
@@ -39,7 +39,10 @@ def __init__(
 
     @classmethod
     def from_url(
-        cls, url: str, connection_kwargs: Optional[Dict[str, Any]] = None, **kwargs: Any
+        cls,
+        url: str,
+        connection_kwargs: Optional[Dict[str, Any]] = {"serverSelectionTimeoutMS": 2000},
```

Comment:
Why you pass this values as default?

Reply:
Yes, I agree. That was bad idea to do like this.